Code/CoreData/RKPropertyInspector+CoreData.m

```diff
@@ -50,7 +50,7 @@ - (NSDictionary *)propertyInspectionForEntity:(NSEntityDescription *)entity
         if ([attributeDescription attributeValueClassName]) {
             Class cls = NSClassFromString([attributeDescription attributeValueClassName]);
             if ([cls isSubclassOfClass:[NSNumber class]] && [attributeDescription attributeType] == NSBooleanAttributeType) {
-                cls = objc_getClass("NSCFBoolean") ?: objc_getClass("__NSCFBoolean") ?: cls;
+                cls = [@YES class];
```

Comment:
maybe we add a helper function somewhere, `RKBooleanClass()` so we don't need to repeat the logic in 3 places?